front/migrations/20250625_findmkdn.ts

```diff
@@ -0,0 +1,55 @@
+import { QueryTypes } from "sequelize";
+
+import { frontSequelize } from "@app/lib/resources/storage";
+import type Logger from "@app/logger/logger";
+import { makeScript } from "@app/scripts/helpers";
+
+const BATCH_SIZE = 10000;
+
+async function countImageMarkdownInMessages(
+  execute: boolean,
+  logger: typeof Logger
+) {
+  logger.info("Processing agent message contents");
+
+  let imageMarkdownCount = 0;
+
+  // Regex to match image markdown: ![anything](anything)
+  const imageMarkdownRegex = /!\[.*?\]\(.*?\)/;
+
+  const messages = await frontSequelize.query<{
+    id: number;
+    content: string;
+  }>(
+    `SELECT amc.id, amc.content
+     FROM agent_message_contents amc
+     WHERE amc.content IS NOT NULL
+       AND amc.content != ''
+     ORDER BY amc.id DESC
+     LIMIT :batchSize`,
+    {
+      replacements: { batchSize: BATCH_SIZE },
```

Comment:
You might need to loop through smaller batch size to get to 10k but maybe it works as is let's give it a try!